corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -88,6 +89,24 @@ def test_duplicate_claim(self):
         self.assertEqual(response.content, 'You have already claimed that case')
 
     @run_with_all_backends
+    def test_duplicate_user_claim(self):
```

Comment:
nit: would be good to test the `get_first_claim` method directly too, even though this seems to serve that purpose.